extensions/hray/src/hray.tsx

```diff
@@ -0,0 +1,94 @@
+import { ActionPanel, Detail, LaunchProps, Action, getPreferenceValues, showToast } from "@raycast/api";
+import { useState, useEffect } from "react";
+import fetch, { HeadersInit } from "node-fetch";
+
+interface CommandArguments {
+  question: string;
+}
+
+interface Preferences {
+  host: string;
+  apiKey?: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x